content/sensu-go/6.1/operations/maintain-sensu/troubleshoot.md

```diff
@@ -379,6 +379,62 @@ Since the `lsb_release` package is not installed, the agent will not be able to
 
 To resolve this problem, install the [`lsb_release` package][8] for your Linux distribution.
 
+## Investigate etcd cluster status
+
+Some issues require investigating the state of the etcd cluster or data stored within etcd. In many such cases, this best accomplished by using the `etcdctl` tool to query and manage the etcd database.
+
+### Obtaining etcdctl
+
+Sensu official packages do not include the etcdctl executable. so you'll need to obtain it from a compatible etcd release.
+
+### Configure etcdctl environment variables
+
+```
```

Comment:
```suggestion
```suggestion
To use etcdctl to investigate etcd cluster and data storage issues, first run these commands to configure etcdctl environment variables:

```
```